packages/code-du-travail-frontend/public/static/modeles.css

```diff
@@ -1,8 +1,8 @@
 .courrier-expediteur {display: flex; align-items: flex-start; flex-direction:column;}
 .courrier-destinataire {display: flex; align-items: flex-end; flex-direction:column;}
 .courrier-signature {display: flex; flex-direction:column; align-items: flex-end;}
-.title-center {display: flex; align-items: center; flex-direction:column; font-size: 1.8rem; font-weight:bold}
+.title-center {display: flex; align-items: center; flex-direction:column; font-size: 1.8rem; font-weight:bold;text-align: center;}
```

Comment:
J'ai rajouté ça à cause de ce comment : https://github.com/SocialGouv/cdtn-docx-to-html/pull/4#discussion_r769423951

Reply:
Bien vu 🕵️‍♀️